corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -674,6 +674,16 @@ def render(self, context):
     return AddToBlockNode(nodelist, 'initial_page_data')
 
 
+@register.simple_tag()
+def set_requirejs_main(request, path):
+    """
+    Indicate that a page should be using RequireJS, by naming the
+    JavaScript module to be used as the page's main entry point.
+    """
+    request.requirejs_main = path
+    return ''
```

Comment:
I think you can change this to
```py
@register.simple_tag(takes_context=True)
def set_requirejs_main(context, path):
    context["request"].requirejs_main = path
    return ''
```
And then you can omit the request argument when you use the tag in a template:
```
{% set_requirejs_main "data_dictionary/js/data_dictionary" %}
```